apps/src/lib/util/StatsigReporter.js

```diff
@@ -125,6 +129,19 @@ class StatsigReporter {
     }
   }
 
+  findOrCreateStableId() {
+    const stableIdKey = 'statsig_stable_id';
+    let stableId = cookies.get(stableIdKey);
+    if (!stableId) {
+      stableId = createUuid();
+      cookies.set(stableIdKey, stableId, {
+        expires: 400,
```

Comment:
Cookie will expire in 400 days:

<img width="1089" alt="Screenshot 2024-07-24 at 9 35 30 AM" src="https://github.com/user-attachments/assets/597dcb9c-ec8a-48ee-8536-66c6f121997a">